locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1446 @@
+# English language text for the bookwyrm UI
+# Copyright (C) 2021 Mouse Reeve
+# This file is distributed under the same license as the bookwyrm package.
+# Mouse Reeve <mousereeve@riseup.net>, 2021
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: 0.1.1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-02-28 08:42-0800\n"
+"PO-Revision-Date: 2021-02-27 13:50+PST\n"
+"Last-Translator: Mouse Reeve <mousereeve@riseup.net>\n"
+"Language-Team: Mouse Reeve <LL@li.org>\n"
+"Language: English \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: bookwyrm/templates/author.html:13 bookwyrm/templates/author.html:14
+msgid "Edit Author"
+msgstr "Modifier l’auteur ou autrice"
+
+#: bookwyrm/templates/author.html:29
+msgid "Wikipedia"
+msgstr "Wikipedia"
+
+#: bookwyrm/templates/author.html:34
+#, python-format
+msgid "Books by %(name)s"
+msgstr Livres par %(name)s
```

Comment:
this is missing quotes

Reply:
This is fixed in https://github.com/mouse-reeve/bookwyrm/pull/665/commits/6e9fc0dbba3ec53274c7a90d202234b204bb9f3f.